.travis.yml

```diff
@@ -4,6 +4,10 @@ compiler:
   - clang
   - gcc
 
+env:
+  - BUILD_TYPE=Release
+  - BUILD_TYPE=Debug
```

Comment:
You can set `CASACORE_CHECK=yes` here also.

So this would look something like this:
```
 - BUILD_TYPE=Release CASACORE_CHECK=no
 - BUILD_TYPE=Debug CASACORE_CHECK=yes
```
which looks much better than the `if` statement below